pymongo/bulk.py

```diff
@@ -315,6 +316,8 @@ def _execute_command(
                     cmd["writeConcern"] = write_concern.document
                 if self.bypass_doc_val:
                     cmd["bypassDocumentValidation"] = True
+                if self.let and cmd_name in ("update", "delete"):
```

Comment:
Do we need to enforce a minwireversion here?

Reply:
That would, I believe, fall under the philosophy of letting the server deliver the error message if possible. For the spec tests for regular 'let' (non-BulkWrite), we rely on server error messages rather than the validation from the driver itself to catch version incompatibility.

Reply:
`cmd_name in ("update", "delete"):` -> `run.op_type in (_DELETE, _UPDATE)` 

We already use run.op_type elsewhere for these kind of checks and it doesn't rely on string comparison.